main/mainworker.cpp

```diff
@@ -13537,6 +13541,8 @@ void MainWorker::HeartbeatCheck()
 
 bool MainWorker::UpdateDevice(const int DevIdx, const int nValue, const std::string &sValue, const std::string &userName, const int signallevel, const int batterylevel, const bool parseTrigger)
 {
+	_log.Debug(DEBUG_NORM, "MainWorker::UpdateDevice: ID: %d", DevIdx);
```

Comment:
Could you remove this debug line (and extra return)

Reply:
Not a necessary part of this change, so removed.